lib/rules/prop-name-casing.js

```diff
@@ -64,6 +65,12 @@ module.exports = {
     schema: [
       {
         enum: allowedCaseOptions
+      },
+      {
+        type: 'object',
+        properties: {
+          ignores: { type: 'array' }
+        }
       }
```

Comment:
Please make the schema more strict:

```suggestion
      {
        type: 'object',
        properties: {
          ignores: {
            type: 'array',
            items: { type: 'string' },
            uniqueItems: true
          }
        },
        additionalProperties: false
      }
```